cumulusci/salesforce_api/filterable_objects.py

```diff
@@ -0,0 +1,260 @@
+NOT_COUNTABLE = (
+    "ContentDocumentLink",  # ContentDocumentLink requires a filter by a single Id on ContentDocumentId or LinkedEntityId
+    "ContentFolderItem",  # Implementation restriction: ContentFolderItem requires a filter by Id or ParentContentFolderId
+    "ContentFolder",  # Similar to above
+    "ContentFolderLink",  # Similar to above
+    "ContentFolderMember",  # Similar to above
+    "IdeaComment",  # you must filter using the following syntax: CommunityId = [single ID],
+    "Vote",  # you must filter using the following syntax: ParentId = [single ID],
+    "RecordActionHistory",  # Gack: 1133111327-118855 (1126216936)
+)
+
+
+NOT_EXTRACTABLE = NOT_COUNTABLE + (
```

Comment:
I'm fine with going forward using this to support the feature, but I think we'll need to revise it before GA. Every object with a non-`Public` sharing model will create a new `__Share` object, so we'll need a different rubric to recognize them dynamically.

Reply:
What do you think about 4726c33945